### PR TITLE
Fix publiccloud/download_repos test module

### PR DESCRIPTION
**Pull request by pdostal**
* Fix `--no-parent` parameter of `wget` - it has no arguments.
 * Fix the URL given to `wget` - it must end with slash.
 * Add debug output to the end of the test module.

- Related error: https://openqa.suse.de/tests/8729394#step/download_repos/81
- Verification run: http://pdostal-server.suse.cz/tests/14783#step/download_repos/104 http://pdostal-server.suse.cz/tests/14776

### CODE REVIEW

File: tests/publiccloud/download_repos.pm [hunk 1]
```diff
@@ -55,7 +55,7 @@ sub run {
             script_run("echo 'Downloading $maintrepo ...' >> ~/repos/qem_download_status.txt");
             my ($parent) = $maintrepo =~ 'https?://(.*)$';
             my ($domain) = $parent =~ '^([a-zA-Z.]*)';
-            $ret = script_run "wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' --domains $domain --no-parent $parent $maintrepo", timeout => 600;
+            $ret = script_run "wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' wget --reject-regex='aarch64\\/|s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/' --domains $domain --no-parent $maintrepo/", timeout => 600;
```
Comment (ilausuch) [inline tests/publiccloud/download_repos.pm:58]: This second wget word is correct here?

Comment (asmorodskyi) [inline tests/publiccloud/download_repos.pm:58]:
```suggestion
            $ret = script_run "wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' --reject-regex='aarch64\\/|s390x\\/|ppc64le\\/|kernel*debuginfo*.rpm|src\\/' --domains $domain --no-parent $maintrepo/", timeout => 600;
```

**Review by ilausuch — APPROVED** [review]

File: tests/publiccloud/download_repos.pm (hunk 1 above)
Comment (pdostal) [reply tests/publiccloud/download_repos.pm]: Just spotted it. It is mistake.

**Review by grisu48 — CHANGES_REQUESTED** [review]
One major issue is that this would break our EC2-ARM test runs.

**Review by asmorodskyi — APPROVED** [review]
LGTM

File: tests/publiccloud/download_repos.pm (hunk 1 above)
Comment (ilausuch) [reply tests/publiccloud/download_repos.pm]:
```
# wget --no-clobber -r -R 'robots.txt,*.ico,*.png,*.gif,*.css,*.js,*.htm*' wget --reject-regex='aarch64\/|s390x\/|ppc64le\/|kernel*debuginfo*.rpm|src\/' --domains download.suse.de --no-parent http://download.suse.de/.....; echo _qmsi-$?-
--2022-05-12 02:53:28--  http://wget/
Resolving wget (wget)... failed: Name or service not known.
```

**Review by grisu48 — APPROVED** [review]
LGTM! Please check that your latest changes are also valid, or manually or by re-scheduling a VR.

**Comment by pdostal** [comment]
Done.

**Pull request merged**